### PR TITLE
Update local setup steps

### PR DESCRIPTION
**Pull request by HonzaCuhel**
Updated README.md local setup steps based on a issue.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,8 +3,33 @@
 This application is used for exporting Yolo V5, V6 and V7 object detection models for OAKs.
 
 ## Running the app locally
-To run the application locally you need to do 2 additional things.
-### Step 1: Updating the `nginx.conf` file
+To run the application locally you need to do these folllowing steps.
+### Step 1: Cloning all repositories
+```
+# Clone tools repository
```
Comment (tersekmatija) [inline README.md:9]: Can't these be cloned with sumodules init and recursive pull? @HonzaCuhel

Comment (HonzaCuhel) [reply README.md:9]: Thanks for the suggestion, will look into it!

Comment (tersekmatija) [reply README.md]: @HonzaCuhel Thanks, can we store the commits of the repo directly instead of having to manually reset them?

Comment (HonzaCuhel) [reply README.md]: @tersekmatija I am not aware of such possibility, I searched and asked ChatGPT

Comment (HonzaCuhel) [reply README.md]: Updated!

**Review by tersekmatija — APPROVED** [review]

**Pull request merged**